WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -50,6 +50,13 @@
             ])
         } %>
       <% end %>
+      <% if current_user && current_user.can_create_competitions? %>
+        <% nav_items << {
+          text: t('.menu.clone'),
+          path: clone_competition_path(@competition),
+          fa_icon: "lock",
```

Comment:
There is a ['clone' icon](http://fontawesome.io/icon/clone/). I'm not making any suggestion, just check it out and use whatever you find cooler =)